### DISCUSSION TITLE
How to know which files will be processed for the current configuration

### DISCUSSION THREAD

**Discussion by C0ZEN** [eslint/eslint#14160] [Q&A]
**Context:**
I just ran the linter on my project and 3 971 files were linted.
I may have a problem regarding the glob because it seems way too much and I wish to know with ease how to list all of those files so that I can check if I need to challenge my configuration to exclude more files.

**Question:**
Do you have an idea how to achieve that easily?
Thanks!

**Comment by mdjermanovic** [comment]
Hi @C0ZEN!

Maybe the bult-in [tap formatter](https://eslint.org/docs/user-guide/formatters/#tap) (add `-f tap` to your lint command) can be helpful.

**Reply by C0ZEN** [reply]
@mdjermanovic it's not but good idea. I do not know the complexity but having a custom formatter to only display all the linted files could be a way! Thanks for the reply

**Reply by mdjermanovic** [reply]
Documentation for custom formatters is here:

https://eslint.org/docs/developer-guide/working-with-custom-formatters

A simple formatter that just displays all linted files:

```js
module.exports = results => results
    .map(({ filePath }) => filePath)
    .join("\n");

```

**Reply by C0ZEN** [reply]
@mdjermanovic thanks! I will take a look today and get back to you.
